sirepo/feature_config.py

```diff
@@ -144,5 +158,20 @@ def b(msg, dev=False):
     x = s.difference(VALID_CODES)
     assert not x, \
         'sim_type(s) invalid={} expected={}'.format(x, VALID_CODES)
+    # SECURITY: _cfg.dynamic_sim_types cannot be validated because their names
+    # cannot be publicly exposed.
+    s.update(_cfg.dynamic_sim_types.keys())
     _cfg.sim_types = frozenset(s)
     return _cfg
+
+
+def _parse_to_dict(string):
```

Comment:
share? pkconfig? But I actually don't see the need.

Reply:
removed 58607069